Makefile

```diff
@@ -4,11 +4,14 @@ SRCDIR=calico_containers
 PYCALICO=$(wildcard $(SRCDIR)/pycalico/*.py)
 BUILD_FILES=Dockerfile build-requirements-frozen.txt
 
+TEST_CONTAINER_FILES=$(shell find calico_test/ -type f ! -name '*.created')
+
 WHEEL_VERSION=0.1.0
 
 default: all
 all: test
 wheel: dist/pycalico-$(WHEEL_VERSION)-py2-none-any.whl
+tester: calico_test.created ## Create the calico/test image
```

Comment:
Add tester to .PHONY (it says PHONEY in this file, so please fix that too!) in line 1.